README.md

```diff
@@ -1,6 +1,10 @@
-jenkins-kubernetes-plugin
+Kubernetes plugin for Jenkins
 =========================
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/role-strategy.svg)](https://plugins.jenkins.io/kubernetes)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/kubernetes-plugin.svg?label=release)](https://github.com/jenkinsci/kubernetes-plugin/releases/latest)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/kubernetes.svg?color=blue)](https://plugins.jenkins.io/kubernetes)
```

Comment:
This image seems broken.

Reply:
It looks good to me

Reply:
Works for me. Requests to shields take a while sometimes, try reloading the page